docs/user/security/reporting.asciidoc

```diff
@@ -32,13 +69,15 @@ POST /_security/user/reporter
 }
 ---------------------------------------------------------------
 
-* If you are using an LDAP or Active Directory realm, you can either assign
+==== If you are using an LDAP or Active Directory realm
```

Comment:
I'm guessing when this was originally written, that Kibana only supported LDAP and Active Directory outside of the built-in `native` realm. We have support for a few other external identity providers now, like `SAML`, `OIDC`, `Kerberos`, and `PKI`. All of these providers would require role mappings instead of the traditional user-role assignment.

What do you think about something like:
```suggestion
==== If you are using an external identity provider, such as LDAP or Active Directory
```